src/dsp/rx_rds.h

```diff
@@ -82,7 +82,7 @@ class rx_rds_store : public gr::block
     void store(pmt::pmt_t msg);
 
     std::mutex d_mutex;
-    boost::circular_buffer<pmt::pmt_t> d_messages;
+    std::queue<pmt::pmt_t> d_messages;
```

Comment:
I presume `circular_buffer` was used (in #226) to ensure that `d_messages` didn't get too large. Is that concern unwarranted? (I think it might be, but I want to be sure before we remove the size limit.)

Reply:
I can't imagine a situation where `MainWindow::rdsTimeout` would be unable to read out all messages fast enough. In that case the main thread would lock up reading RDS messages.
The maximum number of unread messages, I have ever seen is 1: `rx->get_rds_data(buffer, num)` sets `num` to 0 or -1 only.

Reply:
Right, as long as the RDS timer is running while the RDS decoder is (which looks to be the case) this won't be a problem. Thanks for checking!

Reply:
While looking into this, I noticed that `MainWindow::rdsTimeout` has a bug. Fixed in #1050.